### DISCUSSION TITLE
Performance improvements and hardware acceleration

### DISCUSSION THREAD

**Discussion by nathaneltitane** [nathaneltitane/dextop#8] [Ideas]
@RalfWerner - let's discuss here!
@xeffyr - you are welcome to join and share as well :)

**Comment by anonimo82** [comment]
For Hardware acceleration, you can try this:
https://github.com/suhan-paradkar/tewmux-disabled/releases/tag/mesa-hw-v2

**Comment by anonimo82** [comment]
And an idea: use Termux:API to provide device to use in the container (eg. microphone)

**Reply by nathaneltitane** [reply]
@anonimo82 you mean also automate the fetching and installation of the temux api apk? - termux-api package already installs by default. I have access on my devices.

Or did you mean mapping them through the proot command for direct access?

**Reply by anonimo82** [reply]
I mean write "dummy modules" for Kernel, mapping to Termux:API hardware access, to enable hardware inside container for direct access. 

Plesae provide more details on said 'dummy modules' ?
